invite/settings.py

```diff
@@ -3,6 +3,12 @@
 from django.contrib.sites.models import Site
 
 
+INVITE_CONTACT_EMAIL = getattr(
+    settings,
+    'INVITE_CONTACT_EMAIL',
```

Comment:
So what if this were a dict? Something like:
```
INVITE_CONTACT = getattr(
    settings,
    'INVITE_CONTACT',
    {'name': 'Hannah Tarver', 'email': 'hannahtarver@unt.edu'}
```

Reply:
I don't think we need to have Hannah Tarver's information in here as defaults either way. I think it is fine just to have an email address and don't also need a name, though if others feel strongly, a dictionary would be ok. For a default setting value we could possibly use `INVITE_DEFAULT_FROM_EMAIL` i.e. `settings.DEFAULT_FROM_EMAIL` or make something up.

Reply:
Yeah @ldko that sounds better. I don't feel strongly about the dict or including a name, the important thing is the email. And I think defaulting to a local settings value makes sense.

Reply:
hmmm, after looking a bit more, I'm not sure using a *FROM_EMAIL as the default makes a ton of sense here, but I'm also not worried enough about it to feel strongly, just wanted to point out that I don't think those emails are generally going to go to a person or be monitored.

Reply:
yeah, i know. It is just a default as opposed to a dummy one. We could also not provide a default.

Reply:
no big deal either way. _We_ will make sure to provide a value for it :smile: